app/Jobs/CreateCallPowerPostInRogue.php

```diff
@@ -81,6 +81,7 @@ private function getOrCreateUser($mobile)
         if (is_null($user)) {
             $user = gateway('northstar')->asClient()->createUser([
                 'mobile' => $mobile,
+                'sms_status' => 'stop',
```

Comment:
Nothing needed for merge, but It'd be nice to add all of these constants like this one, and the post source/details as a new array items in the [config](https://github.com/DoSomething/chompy/blob/master/config/import.php), e.g. `config('call_power.user_sms_status`)`, `config('call_power.post.source')`, `type`, etc

Reply:
Love that idea, @aaronschachter ! Made a card for that [here](https://www.pivotaltracker.com/n/projects/2019429/stories/164932452).